tests/import_schemas.py

```diff
@@ -0,0 +1,213 @@
+"""
+A script to generate the schemas for the GA4GH protocol
+"""
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import glob
+import shutil
+import os.path
+import tarfile
+import tempfile
+import re
+
+import avro.schema
+
+import utils
+
+
+class SchemaClass(object):
+    """
+    Class to convert an Avro JSON definition of a GA4GH type into the
+    corresponding Python class.
+    """
+    def __init__(self, sourceFile):
+        self.sourceFile = sourceFile
+        with open(sourceFile) as sf:
+            self.schemaSource = sf.read()
+            self.schema = avro.schema.parse(self.schemaSource)
+        self.name = self.schema.name
+
+    def getFields(self):
+        """
+        Returns the list of avro fields sorted in order of name.
+        """
+        return sorted(self.schema.fields, key=lambda f: f.name)
+
+    def isSearchRequest(self):
+        """
+        Returns True if the class we are converting is a subclass of
+        SearchRequest, and False otherwise.
+        """
+        return re.search('Search.+Request', self.name) is not None
+
+    def isSearchResponse(self):
+        """
+        Returns True if the class we are converting is a subclass of
+        SearchResponse, and False otherwise.
+        """
+        return re.search('Search.+Response', self.name) is not None
+
+    def getValueListName(self):
+        """
+        Returns the name of the list used to store values in a page for
+        a SearchRequest subclass.
+        """
+        assert self.isSearchResponse()
+        names = [field.name for field in self.getFields()]
+        # We assume that there are exactly two fields in every
+        # SearchResponse: nextPageToken and the value list.
+        names.remove('nextPageToken')
+        assert len(names) == 1
+        return names[0]
+
+    def getEmbeddedTypes(self):
+        """
+        Returns the set of embedded types in this class.
+        """
+        # TODO need to clarify how we operate on Unions here. The current
```

Comment:
We should remove this comment, and address it. What do we assume about the structure of unions in v0.6?